extensions/nginx/index.js

```diff
@@ -135,46 +127,45 @@ class NginxExtension extends Extension {
 
         return this.ui.listr([{
             title: 'Checking DNS resolution',
-            task: () => Promise.fromNode(cb => dns.lookup(parsedUrl.hostname, {family: 4}, cb)).catch((error) => {
-                if (error.code !== 'ENOTFOUND') {
-                    // Some other error
-                    return Promise.reject(new CliError({
-                        message: `Error trying to lookup DNS for '${parsedUrl.hostname}'`,
-                        err: error
-                    }));
+            task: async () => {
```

Comment:
Is it worth using async / await here? Feels like promise / catch works better in this scenario

Reply:
yeah, guess it doesn't really add too much